hack/verify-codecgen.sh

```diff
@@ -102,6 +104,8 @@ for current in ${index[@]}; do
   sed 's/YEAR/2015/' "${initial_dir}/hack/boilerplate/boilerplate.go.txt" > "${base_generated_file}.tmp"
   cat "${base_generated_file}.1tmp" >> "${base_generated_file}.tmp"
   rm "${base_generated_file}.1tmp"
+  # restore the generated file
```

Comment:
Sure - it seems I must have been really sleepy when writing this comment.
Yeah - what you are doing seems ok.